controllers/storagecluster/quickstart_constants.go

```diff
@@ -50,16 +50,17 @@ spec:
 
    3. Click the Action menu (⋮)-> Add Storage
 
-   4. Select \"Create mew claim\". You would select \"Use existing claim\" here if you wanted to restore an existing persistent volume to a redeployed application.
+   4. Select \"Create new claim\". You would select \"Use existing claim\" here if you wanted to restore an existing persistent volume to a redeployed application.
 
-   5. Select the appropriate type of storage for your application/
-
-    - **For block storage**, select ocs-storagecluster-ceph-rbd.
-
-    - **For file storage**, select ocs-storagecluster-cephfs.
+   5. Select the appropriate type of storage for your application.
 
+      **For block storage**, select ocs-storagecluster-ceph-rbd.
+   
+      **For file storage**, select ocs-storagecluster-cephfs.
+  
+   
```

Comment:
Why 2 new lines? Wouldn't 1 suffice?

Reply:
having two line keeps the points segregated nicely. having one line outputs it like this 
![Screenshot from 2021-01-12 19-29-06](https://user-images.githubusercontent.com/57935785/104323757-91432f80-550c-11eb-8dcf-5769d9597f84.png)